src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -6904,7 +6957,27 @@ extends System.Object
         }
 
         [Fact]
-        public void ToString_BadBase_SealedToString()
+        public void ToString_NewToString_SealedBaseToString()
+        {
+            var source = @"
+public record A
+{
+    public sealed override string ToString() => throw null;
+}
+
+public record B : A
+{
+    public new string ToString() => throw null;
+}";
+
+            var comp = CreateCompilation(source, parseOptions: TestOptions.RegularPreview);
+            comp.VerifyEmitDiagnostics();
```

Comment:
>comp.VerifyEmitDiagnostics(); [](start = 12, length = 29)

For success scenarios like this we prefer to compile, execute the compiled code and observe that it behaves as expected at runtime. For example in this case, it would be good to verify that calling B.ToString calls the method from B, but calling ((A)B).ToString calls the method from A. The methods can return distinct values and we can print them and verify the output.

Reply:
OK, I'll change it

Reply:
Done (also in ToString_BadBase_SealedToString)